tests/components/Forms/JobSearchForm.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
```

Comment:
Great job on covering the happy path can we cover the sad path, how does the component handle failures?